src/main/java/com/squareup/kotlinpoet/TypeName.kt

```diff
@@ -216,6 +216,7 @@ abstract class TypeName internal constructor(
 @JvmField val CHAR = ClassName("kotlin", "Char")
 @JvmField val FLOAT = ClassName("kotlin", "Float")
 @JvmField val DOUBLE = ClassName("kotlin", "Double")
+@JvmField val DYNAMIC = ClassName("", "dynamic")
 
```

Comment:
I don't think it makes sense to model this as a `ClassName`. None of the methods on `ClassName` are relevant for it.

Reply:
Agree. What about `object Dynamic : TypeName()`?

Reply:
Works for me.